clang/test/Sema/aarch64-sme-func-attrs-without-target-feature.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -fsyntax-only -verify %s
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -std=c++23 -fsyntax-only -verify %s
```

Comment:
What about this change requires c++23?

Reply:
I think adding attributes to lambdas before the parameter list was only allowed in C++23. If I don't pass this flag, the error emitted is instead `"'attribute' in this position is a C++23 extension"`.